stdlib/uniformTransforms.hk

```diff
@@ -0,0 +1,29 @@
+# Hakaru Standard Library
+# author: Mahmoud Khattab
+
+# Uniform distribution transformations
+
+######################
+# Uniform Transforms #
+######################
+
+# Pareto Distribution.
+def pareto(lambda prob, kappa prob):
+	X <~ uniform(0,1)
+	return lambda / (real2prob(X) ** (1/kappa))
+
+# Standard power distribution
+def standardPower(beta prob):
+	X <~ uniform(0,1)
+	return real2prob(X) ** (1/beta)
+
+# Gompertz distribution
+# Waiting on log to be fixed to test this
+def gompertz(delta prob, kappa prob):
+	X <~ uniform(0,1)
+	return log(real2prob(log(real2prob(X))*log(kappa)/delta))/log(kappa)
```

Comment:
Where this log(..) line come from? I am unfamiliar with it.

Reply:
http://www.math.wm.edu/~leemis/chart/UDR/UDR.html

See the transformation from Standard Uniform -> Gompertz